src/NServiceBus.Transport.SQS/Extensions/MessageExtensions.cs

```diff
@@ -15,7 +15,14 @@ static class MessageExtensions
         {
             if (string.IsNullOrEmpty(transportMessage.S3BodyKey))
             {
-                return Convert.FromBase64String(transportMessage.Body);
+                if (transportMessage.Body == TransportMessage.EmptyMessage)
+                {
+                    return System.Text.Encoding.Default.GetBytes("");
```

Comment:
As we discussed during the sync, we should evaluate if this needs to be `UTF8` instead of `Default`

Reply:
I'm not entirely sure if we have to use UTF8 or if we can use `Unicode`

https://docs.aws.amazon.com/AWSSimpleQueueService/latest/APIReference/API_SendMessage.html

links to https://www.w3.org/TR/REC-xml/#charsets

which says

> The mechanism for encoding character code points into bit patterns may vary from entity to entity. All XML processors MUST accept the UTF-8 and UTF-16 encodings of Unicode

Reply:
That being said I think for this PR the discussion is not relevant. See my other comment